docs/source/configuration.md

```diff
@@ -116,7 +159,7 @@ Following fields are required (and can be customized) in the [`challenge_config.
 
   - **dataset_split_id**: Id of `dataset_split`
 
-  - **visibility**: It will set the visibility of the numbers corresponding to metrics for this `challenge_phase_split`. Select one of the following positive integers based on the visibility level you want:
+  - **visibility**: It will set the visibility of the numbers corresponding to metrics for this `challenge_phase_split`. Select one of the following positive integers based on the visibility level you want:(Optional Default is `3`)
```

Comment:
Add comma after optional.

Also move the table below here if it renders properly.